ambry-rest/src/test/java/com.github.ambry.rest/NettyResponseChannelTest.java

```diff
@@ -392,6 +392,23 @@ public void keepAliveTest() {
     channel.close();
   }
 
+  /**
+   * Tests that the underlying network channel is closed when {@link NettyResponseChannel#close()} is called.
+   */
+  @Test
+  public void closeTest() {
```

Comment:
This test class is a little complex. If you look that URI in the request and go down to `MockNettyMessageProcessor` and find the correct switch case, you will see the execution of the test. We can discuss this if you want more context.